operations/firmware/FAS_Recipes.md

```diff
@@ -1,23 +1,25 @@
-# Recipes
+# FAS Recipes
 
-> **`NOTE`** This file contains a collection of various FAS recipes for performing updates.
-For step by step directions and commands, see [FAS_Use_Cases](FAS_Use_Cases.md)
+> **`NOTE`** This is a collection of various FAS recipes for performing updates.
+> For step by step directions and commands, see [FAS Use Cases](FAS_Use_Cases.md).
 
 The following example JSON files are useful to reference when updating specific hardware components. In all of these examples, the `overrideDryrun` field will be set to `false`; set them to `true` to perform a live update.
 
-When updating an entire system, walk down the device hierarchy component type by component type, starting first with Routers (switches), proceeding to Chassis, and then finally to Nodes. While this is not strictly necessary, it does help eliminate confusion.
+When updating an entire system, walk down the device hierarchy component type by component type, starting first with routers (switches), proceeding to chassis, and then finally to nodes.
+While this is not strictly necessary, it does help eliminate confusion.
 
 Refer to [FAS Filters](FAS_Filters.md) for more information on the content used in the example JSON files.
 
----
 ## Manufacturer : Cray
 
-### Device Type: ChassisBMC | Target: BMC
+### Cray: Device Type: `ChassisBMC` | Target: BMC
```

Comment:
```suggestion
### (Cray) Device Type: `ChassisBMC` | Target: BMC
```
I would recommend not using the `:` character after all of the manufacturer names here. It makes it a little confusing to read with how device type and target are formatted.